vizro-ai/docs/pages/user-guides/add-generated-chart-usecase.md

```diff
@@ -119,13 +121,14 @@ This object must then be passed to the `figure` argument of the Vizro [Graph](ht
 
 We can also use Vizro-AI dynamically and assign the output of `plot()` directly to the fig variable, enabling its reuse in the `vm.Graph.figure` argument.
 This method offers streamlined efficiency, eliminating the need for code copying.
-However, it is important to note that each dashboard run triggers an API call to OpenAI, possibly escalating costs, although this can be avoided if the `fig` object is stored and retrieved as needed, instead of making repeated calls to `plot()`.
+Note that each dashboard run triggers an API call to OpenAI, possibly escalating costs. This can be avoided if the `fig` object is stored and retrieved as needed, instead of making repeated calls to `plot()`.
```

Comment:
```suggestion
Note that each dashboard run triggers an API call to the LLM, possibly escalating costs. This can be avoided if the `fig` object is stored and retrieved as needed, instead of making repeated calls to `plot()`.
```